lib/plugins/helper/full_url_for.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+const { full_url_for } = require('hexo-util');
+
+module.exports = function(path = '/') {
```

Comment:
can be just `path`, hexo-util.full_url_for already set default parameter.

Reply:
Fixed, and related test has been added.